src/state/apis/zapper/zapperApi.ts

```diff
@@ -485,49 +513,41 @@ export const zapper = createApi({
-                              .toString()
-                          : undefined
-                      const token0PoolRatio =
-                        token0ReservesBaseUnit && totalSupplyBaseUnit
-                          ? token0ReservesBaseUnit.div(totalSupplyBaseUnit).toString()
-                          : undefined
-                      const token1PoolRatio =
-                        token1ReservesBaseUnit && totalSupplyBaseUnit
-                          ? token1ReservesBaseUnit.div(totalSupplyBaseUnit).toString()
-                          : undefined
-
-                      if (!(token0PoolRatio && token1PoolRatio)) return
-
-                      return [
-                        toBaseUnit(token0PoolRatio.toString(), asset.tokens[0].decimals),
-                        toBaseUnit(token1PoolRatio.toString(), asset.tokens[1].decimals),
-                      ] as const
-                    }
+                    const token0ReservesCryptoPrecision = asset.dataProps?.reserves?.[0]
+                    const token1ReservesCryptoPrecision = asset.dataProps?.reserves?.[1]
+                    const token0ReservesBaseUnit =
+                      typeof token0ReservesCryptoPrecision === 'number'
+                        ? bnOrZero(
+                            bnOrZero(bnOrZero(token0ReservesCryptoPrecision.toFixed()).toString()),
+                          ).times(bn(10).pow(asset.decimals ?? 18))
+                        : undefined
+                    const token1ReservesBaseUnit =
+                      typeof token1ReservesCryptoPrecision === 'number'
+                        ? bnOrZero(
+                            bnOrZero(bnOrZero(token1ReservesCryptoPrecision.toFixed()).toString()),
+                          ).times(bn(10).pow(asset.decimals ?? 18))
+                        : undefined
+                    const totalSupplyBaseUnit =
+                      typeof asset.supply === 'number'
+                        ? bnOrZero(asset.supply)
+                            .times(bn(10).pow(asset.decimals ?? 18))
+                            .toString()
+                        : undefined
+                    const token0PoolRatio =
+                      token0ReservesBaseUnit && totalSupplyBaseUnit
+                        ? token0ReservesBaseUnit.div(totalSupplyBaseUnit).toString()
+                        : undefined
+                    const token1PoolRatio =
+                      token1ReservesBaseUnit && totalSupplyBaseUnit
+                        ? token1ReservesBaseUnit.div(totalSupplyBaseUnit).toString()
+                        : undefined
+
+                    if (!(token0PoolRatio && token1PoolRatio)) return
+
+                    return [
+                      toBaseUnit(token0PoolRatio.toString(), asset.tokens[0].decimals),
+                      toBaseUnit(token1PoolRatio.toString(), asset.tokens[1].decimals),
+                    ] as const
```

Comment:
This didn't change, only the indent did because of the `if (asset.type === 'app-token')` removal